farm/data_handler/processor.py

```diff
@@ -730,6 +542,8 @@ def __init__(
         :type doc_stride: int
         :param max_query_length: Maximum length of the question (in number of subword tokens)
         :type max_query_length: int
+        :param kwargs: placeholder for passing generic parameters
+        :type kwargs: object
```

Comment:
- we should move label_list into the constructor as an arg with default values
 - we don't need to set dev_filename etc here (it's passed to the parent class and gets set there)